### ISSUE TITLE
Standin is affecting the Stage rendering

### ISSUE THREAD

**Issue by Xaia** [Autodesk/maya-usd#467]
**Describe the bug**
Turns out that if you have Standin hidden (and disabled lights operator) those Standin lights are affecting Stage.

**Steps to reproduce**
1. Load Nvidia Attic standin, turn off lights with operator, hide the standin
2. bring nvidia Attic Stage, kill all the lights inside the Stage and render

**Expected behavior**
Hidden standin should not affect anything

**Attachments**

![5OyAPQnfxj](https://user-images.githubusercontent.com/60515654/80390142-52047e00-88ac-11ea-8d43-e161e0adbf91.gif)

**Specs (if applicable):**
 -Windows 10
- Maya PR114

**Comment by santosd** [comment]
Hi @Xaia I am trying this out on our latest build and for me when I initially open the scene, without making any changes everything is washed out on render. If I delete the USD lights and camera it is all still the same washed out look. If I add my own lights it doesn't change. Wondering if there are any extra steps you are taking to get the proper render?

**Comment by Xaia** [comment]
I do not believe I did. For sure I had both Stage and Standin with disabled lights in the scene. 
I had it render "correctly" here. Try to disable the lights before you delete them.
https://github.com/Autodesk/maya-usd/issues/466

**Comment by santosd** [comment]
Even disabling the lights before deleting didn't help on my end. I will keep testing to see if I can get it to work. I am seeing only the extremely washed out render on my end.

**Comment by santosd** [comment]
Hi @Xaia, I have spent some more time recently taking a look at this and I was still not able to reproduce the issue you are seeing on my end. I am still having issues in which the whole scene is always washed out.

Here is what I tried:

1. Create a stage from the NVIDIA Attic scene.
2. Delete the lights in the stage.
3. Render with Arnold and everything is washed out.

I have also tried with and aiStandIn and everything is still washed out.

I am wondering:

1. Are you still seeing this issue on your end?
2. Are you using a specific version of the file that is meant for rendering with Arnold?
3. Can you provide a video showing your workflow?
It's possible that I am not doing the exact same workflow as you and I think a video would be extremely helpful.

**Comment by santosd** [comment]
Hi @Xaia,
Just checking back to see if you could provide some further information from my above comment. If I don't hear back, since I am having trouble reproducing the issue I will go ahead and close the ticket out for the time being.
Thank you and looking forward to hearing back.

**Comment by Xaia** [comment]
Hi,

I am away from my PC for a while, I'll will let you know once I am back.

Cheers
